app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/mnemonics/RestoreMnemonicsScreen.kt

```diff
@@ -173,24 +173,51 @@ private fun RestoreMnemonicsContent(
                     }
                 )
             } else {
-                RadixPrimaryButton(
+                Column(
                     modifier = Modifier
                         .fillMaxWidth()
-                        .imePadding()
-                        .padding(RadixTheme.dimensions.paddingDefault),
-                    text = stringResource(
-                        when (state.screenType) {
-                            RestoreMnemonicsViewModel.State.ScreenType.Entities -> R.string.recoverSeedPhrase_enterButton
-                            RestoreMnemonicsViewModel.State.ScreenType.SeedPhrase -> R.string.common_continue
-                            RestoreMnemonicsViewModel.State.ScreenType.NoMainSeedPhrase ->
-                                R.string.recoverSeedPhrase_skipMainSeedPhraseButton
+                        .padding(top = RadixTheme.dimensions.paddingMedium)
+                ) {
+                    if (state.screenType is RestoreMnemonicsViewModel.State.ScreenType.Entities) {
+                        if (!state.isMainBabylonSeedPhrase) {
+                            RadixTextButton(
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .padding(horizontal = RadixTheme.dimensions.paddingLarge),
+                                text = stringResource(id = R.string.recoverSeedPhrase_skipButton),
+                                onClick = onSkipSeedPhraseClick
+                            )
                         }
-                    ),
-                    enabled = state.screenType != RestoreMnemonicsViewModel.State.ScreenType.SeedPhrase ||
-                        state.seedPhraseState.seedPhraseValid,
-                    isLoading = state.isRestoring,
-                    onClick = onSubmitClick
-                )
+
+                        if (state.isMainBabylonSeedPhrase && state.isMandatory.not()) {
+                            RadixTextButton(
+                                modifier = Modifier
+                                    .fillMaxWidth(),
```

Comment:
Instead you can attach this padding here.

Reply:
In both buttons you mean ?

Reply:
This button has no padding with it but the button above has. That is why I am suggesting adding the same paddiing here as above and removing it from the column.